app/graphql/mutations/create_in_progress_submission_report.rb

```diff
@@ -0,0 +1,46 @@
+module Mutations
+  class CreateInProgressSubmissionReport < ApplicationQuery
```

Comment:
```suggestion
  class BeginProcessingSubmissionReport < ApplicationQuery
```

I'm not too happy with the replacement name I've written above. 🤔

Reply:
Couldn't find a better one either 😬